### DISCUSSION TITLE
v1.10.2

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#487] [Announcements]
Big thanks to [Yasin Baran](https://github.com/brnysn) for his first contribution to LinkAce! 🥳

- Some websites are now properly checked for meta information ([#465](https://github.com/Kovah/LinkAce/issues/465))
- Automatic app backups should now run properly without permission changes ([#475](https://github.com/Kovah/LinkAce/issues/475))
- Timeouts for saving links to the Wayback Machine are now ignored ([#472](https://github.com/Kovah/LinkAce/issues/472))
- Added new configuration options for S3 for third-party services, see [advanced .env settings](https://www.linkace.org/docs/v1/configuration/env-settings/) for more details. ([#476](https://github.com/Kovah/LinkAce/issues/476))
- LinkAce skips the link check in the import command if email is not configured ([#445](https://github.com/Kovah/LinkAce/issues/445))
- Remove the typed characters from tag and list select once item is selected ([#378](https://github.com/Kovah/LinkAce/issues/378))
- Some websites are now properly checked for meta information ([#465](https://github.com/Kovah/LinkAce/issues/465))
- Updated dependencies

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v1.10.2'>v1.10.2</a>.</em>

**Comment by gavinr** [comment]
Thanks! Some great fixes/updates in this release!

**Comment by xanthos84** [comment]
hi,
i have used your docker update script,
and after doing its thing with no errors - linkace started with an empty db? i cant login any more because my user is not there.
of course i can now search my backup and reverse everything,
but it is weird that it happened?

i have pasted the log from the update procedure - the db update failed, but when I ran it afterwards manually, it told me nothing to upgrade.

**Comment by xanthos84** [comment]
`
==============================
 LinkAce Docker Update Script
==============================

This script assumes you are using the standard Docker setup for Linkace. It is only save to use if you did not modify any Docker-related files.
Please make a backup of your database before you continue.

Do you want to upgrade LinkAce now? [y/n] y
> Deleting the application container volume...
Stopping linkace_nginx_1 ... done
Stopping linkace_app_1   ... done
Stopping linkace_db_1    ... done
Stopping linkace_redis_1 ... done
Removing linkace_nginx_1 ... done
Removing linkace_app_1   ... done
Removing linkace_db_1    ... done
Removing linkace_redis_1 ... done
Removing network linkace_default

> Deleting the application container volume...
linkace_linkace_app

> Pulling latest LinkAce image from the Docker Hub...
Using default tag: latest
latest: Pulling from linkace/linkace
2408cc74d12b: Pull complete
fde5ea1cb188: Pull complete
3935ba60366a: Pull complete
4712e34f1d29: Pull complete
6a3b65d24407: Pull complete
25e8a2ef4024: Pull complete
a7cac4a2dc31: Pull complete
ec9e31a6a1c5: Pull complete
847d29eeaaba: Pull complete
3619c690d25f: Pull complete
4cb87711e3ad: Pull complete
af9867338417: Pull complete
66d305d31e1c: Pull complete
4c15bb013c57: Pull complete
15f32d855887: Pull complete
1ab9e717225d: Pull complete
4b334f752095: Pull complete
d92af8ee34b2: Pull complete
06fcb4ff67f2: Pull complete
dbaa03f2a3ac: Pull complete
40c74bfe53cc: Pull complete
389d57db0375: Pull complete
bd1dac8913ac: Pull complete
a7b395a3b332: Pull complete
1c4654be9316: Pull complete
0a244df5b066: Pull complete
324eecaf401e: Pull complete
083946991313: Pull complete
ebb22d8f647c: Pull complete
459a4e57c239: Pull complete
a60923ee6a1b: Pull complete
0a6737360b94: Pull complete
Digest: sha256:9c9ef5b267801c462387118c20a6a04ec2929b4f94be9133ff065f65a57e55b4
Status: Downloaded newer image for linkace/linkace:latest
docker.io/linkace/linkace:latest

> Restarting the application...
Creating network "linkace_default" with the default driver
Creating volume "linkace_linkace_app" with default driver
Creating volume "linkace_linkace_logs" with default driver
Creating volume "linkace_db" with local driver
Creating linkace_db_1    ... done
Creating linkace_redis_1 ... done
Creating linkace_app_1   ... done
Creating linkace_nginx_1 ... done

The LinkAce image was successfully updated.

You should upgrade the database now. Should the script take care of it? [y/n] y
> Migrating database...
ERROR: No such service: php

LinkAce was upgraded successfully!

Should the configuration and routes be cached now? This might improve overall performance. [y/n] n`

**Reply by Kovah** [reply]
The script should not automatically delete any database volumes while upgrading, but it's been a while when I last tested the script.
`ERROR: No such service: php` is weird, do you have the latest version of the script?

**Comment by xanthos84** [comment]
yes i have the latest version,
for me the command should be
"docker exec -it linkace_app_1 php artisan migrate"
because that is what i have executed afterwards my restore + re-update
and then the update worked out well.
